### DISCUSSION TITLE
Facebook login not working when using responseType: code

### DISCUSSION THREAD

**Discussion by alucardu** [nuxt-community/auth-module#1566] [Support Requests]
### Version
module: 5.0.0-1611574754
nuxt: 2.14.12

#### [mode](https://nuxtjs.org/api/configuration-mode): 
 - [ X ] universal
 - [ ] spa

### Nuxt configuration
```
    auth: {
      redirect: {
        login: '/',
        logout: '/',
        callback: '/',
        home: '/'
      },
      plugins: [
        // { src: '~plugins/axios' }
      ],
      strategies: {
        facebook: {
          clientId: '1303838433331208',
          scope: ['public_profile', 'email'],
          // responseType: 'code'
        },
      }
    },
```

## Reproduction
https://codesandbox.io/s/sad-frog-wtrml

#### What is expected?
Login using the Facebook provider and getting the Facebook code.
#### What is actually happening?
Nuxt auth is not logging in.
#### Steps to reproduce
1. Go to the sandbox: https://codesandbox.io/s/sad-frog-wtrml
2. Click the `Facebook` button.
3. Refresh the page and see that the status is still false.

To resolve the issue:
1. Open nuxt.config.js
2. Comment out line #39
3. Take the above steps and see the status turn to true (after a refresh)

## Checklist
* [ X ]  I have tested with the latest Nuxt version and the issue still occurs
* [ X ]  I have tested with the latest module version and the issue still occurs
* [ X ]  I have searched the issue tracker and this issue hasn't been reported yet

I've also created a sandbox with the older version of Nuxt Auth and getting the same issue: https://codesandbox.io/s/festive-tesla-crht7

**Comment by alucardu** [comment]
Any chance someone taking a look at this issue?

**Comment by KHSR650** [comment]
I think there was no chance. Though we are trying to update the default options for facebook provider the default options won't update to skip to get the profile data from access token. By default facebook provider is hitting to get profile from any response for both code and token. If we apply code the profile fetch api will failed then is shows complete auth failed.
Better to avoid using facebook provider if we need code.
